libcaja-private/caja-directory.c

```diff
@@ -522,9 +514,12 @@ caja_directory_are_all_files_seen (CajaDirectory *directory)
 {
     g_return_val_if_fail (CAJA_IS_DIRECTORY (directory), FALSE);
 
-    return EEL_CALL_METHOD_WITH_RETURN_VALUE
-           (CAJA_DIRECTORY_CLASS, directory,
-            are_all_files_seen, (directory));
+    if (CAJA_DIRECTORY_GET_CLASS(directory)->are_all_files_seen == NULL)
+    {
+        return FALSE;
+    } else {
+        return CAJA_DIRECTORY_GET_CLASS(directory)->are_all_files_seen (directory);
+    }
```

Comment:
brace style

Reply:
Can you give me an example? What kind of style will be better, I really don't know, thank you.

Reply:
Look at the code around; apart from a few inconsistencies, it's putting braces on their own lines, e.g.

```c
    if (CAJA_DIRECTORY_GET_CLASS(directory)->are_all_files_seen == NULL)
    {
        return FALSE;
    }
    else
    {
        return CAJA_DIRECTORY_GET_CLASS(directory)->are_all_files_seen (directory);
    }
```